lib/linter.js

```diff
@@ -135,7 +133,8 @@ class Linter {
 
     let loadedRules = this.config.loadedRules;
     for (let ruleName in configuredRules) {
-      if (configuredRules[ruleName] === false) {
+      const configuredRule = configuredRules[ruleName];
+      if (configuredRule === false || (configuredRule && configuredRule.config === false)) {
```

Comment:
We should not need to check both formats, but we should probably leverage severity here (0 being off, and would replace the check for `false` that we have currently)

Reply:
same as above :(